docs/distributed/POOLS.md

```diff
@@ -0,0 +1,52 @@
+## Decommissioning
+
+### How to decommission a pool?
+```
+λ mc admin decommission start alias/ http://minio{1...2}/data{1...4}
+```
+
+### Status decommissioning a pool
+
+#### Decommissioning without args lists all pools
+```
+λ mc admin decommission status alias/
+┌─────┬─────────────────────────────────┬──────────────────────────────────┬────────┐
+│ ID  │ Pools                           │ Capacity                         │ Status │
+│ 1st │ http://minio{1...2}/data{1...4} │ 439 GiB (used) / 561 GiB (total) │ Active │
+│ 2nd │ http://minio{3...4}/data{1...4} │ 329 GiB (used) / 421 GiB (total) │ Active │
+└─────┴─────────────────────────────────┴──────────────────────────────────┴────────┘
+```
+
+#### Decommissioning status
+```
+λ mc admin decommission status alias/ http://minio{1...2}/data{1...4}
+Progress: ===================> [1GiB/sec] [15%] [4TiB/50TiB]
+Time Remaining: 4 hours (started 3 hours ago)
+```
+
+#### A pool not under decommissioning will throw an error
+```
+λ mc admin decommission status alias/ http://minio{1...2}/data{1...4}
+ERROR: This pool is not scheduled for decommissioning currently.
+```
+
+### Canceling a decommission?
```

Comment:
I don't really understand what "canceled" state means. What does it do? Is it a "pause" function?

Reply:
It's more like stop the decommission for now.

Reply:
Couldn't we name it "pause", if that is what we want to do?

Reply:
Yeah well @klauspost I thought cancel is what happens more than pause and there is no resume. You just have to start again what you did before, if necessary it will resume.

Just don't want to add anymore CLI. Let me add more details..